addons/web/static/src/views/form/form_controller.scss

```diff
@@ -240,9 +240,13 @@
     // No sheet
     &.o_form_nosheet, .o_form_nosheet {
         @include o-webclient-padding($top: $o-sheet-vpadding, $bottom: $o-sheet-vpadding);
+        background-color: var(--formView-sheet-background);
 
         .o_form_statusbar {
             margin: (-$o-sheet-vpadding) (-$o-horizontal-padding) $o-sheet-vpadding (-$o-horizontal-padding);
+            background-color: map-get($grays, '100');
+            border-bottom: $border-width solid $border-color;
+            padding: map-get($spacers, 2) map-get($spacers, 3) !important;
```

Comment:
Be careful since formView's paddings are now responsive.
You should use CSS variables (eg. `--formView-sheet-padding-x`)... could you check?

Reply:
I did not notice a difference since the variables used also use the spacers map 



but again, using the syntax you suggest allows us to understand easily how we control the padding of the element and it will make any change on that values smoother.